src/views/pages/product/single.twig

```diff
@@ -357,6 +357,12 @@
                             <div class="out-of-stock min-h-7 leading-7 hidden text-base text-red-600 !opacity-50 font-bold">{{ trans('pages.products.out_of_stock')}}</div>
                         </div>
                     </section>
+					{% if product.has_preorder_campaign and (product.preorder.availability_date or product.preorder.end_date) %}
+                        <section class="mt-5 p-5">
+							<salla-count-down pre-order-date="{{ product.preorder.end_date  }}" pre-order-availability-date="{{ product.preorder.availability_date }}" boxed="true" labeled="true"></salla-count-down>
+						</section>
+                    {% endif %}
```

Comment:
- please make this `has_preorder_campaign` from backend means `has_preorder_campaign && (product.preorder.availability_date or product.preorder.end_date)` to be straightforward and simple for public usage

- make `preorder` prop, accept json then pass to it `pre-order="{{product.preorder}}"` 

- no need to pass "true" just pass `<salla-count-down boxed labeled>`